### DISCUSSION TITLE
Gauntlet Legends N64 on xbox series X , sound popping noise

### DISCUSSION THREAD

**Discussion by Beardedbastard96** [gonetz/GLideN64#2882] [Q&A]
I am using Angry lion to emulate on my xbox series X , and the problem is that it fixed the black borders which I can't fix using GlideN64 , but at the cost of sound quality.

What suggestion do you guys have? Im using Nintendo 64 powered by libretto v 1.15.0.6 with Mugenplus-Next 2.4-Gles 0050d35

**Comment by weinerschnitzel** [comment]
Some thoughts:

1. Gauntlet legends is a Musyx game and Musyx has been HLE'd for some years now. Try HLE rsp plugin if you have been using a low level rsp plugin. The SDL audio plugin has some settings to change buffer sizes that I have seen help games like 007 TWINE, specifically on og xbox. You may have to edit source code and rebuild that plugin.
2. Xbox libretro port uses a wrapper to convert gles to directx which could slow down emulation and result in stutters and pops. Try turning settings down. I.E. try lower resolution, legacy blending, inaccurate texture coordinates etc.

You may also get some better insight on libretro forums.

Note: this advise is for using gliden64, since you posted here. If you have stutters with angrylion/parallel rdp, there is not much you can do to speed it up other than using different  plugins.
